src/AddTodoForm.jsx

```diff
@@ -1,8 +1,19 @@
-function AddTodoForm() {
+function AddTodoForm(props) {
+    let {onAddTodo} = props;
```

Comment:
Just a tip: always declare variables with `const` if they are not supposed to change.